### ISSUE TITLE
Connect FIlter Menu City Council Districts dropdown to API

### ISSUE THREAD

**Issue by TylerMatteo** [NYCPlanning/ae-cp-map#10]
## Acceptance Criteria
- [ ] Populate options in City Council Districts dropdown in FilterMenu component ([Figma](https://www.figma.com/design/LYHHoPop9l0jpEivk5CFzJ/Capital-Projects-Portal?node-id=251-6258&t=i9OdRPOzwaKcOnqa-0)) using data from `findCityCouncilDistricts` endpoint.

**Comment by TangoYankee** [comment]
Blocked by: #18

**Comment by TylerMatteo** [comment]
Closing in favor of #28
